xblockutils/public/studio_container.js

```diff
@@ -1,20 +1,21 @@
 function StudioContainerXBlockWithNestedXBlocksMixin(runtime, element) {
     var $buttons = $(".add-xblock-component-button", element),
+        $view = $('.add-xblock-component', element),
```

Comment:
@itsjeyd naming this one is hard :P. I'll change it to `$addComponent` in the next patch.